src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -106,28 +107,16 @@
     <Compile Include="System\Net\Http\Headers\UriHeaderParser.cs" />
     <Compile Include="System\Net\Http\Headers\ViaHeaderValue.cs" />
     <Compile Include="System\Net\Http\Headers\WarningHeaderValue.cs" />
-    <Compile Include="$(CommonPath)System\IO\StreamHelpers.CopyValidation.cs"
-             Link="Common\System\IO\StreamHelpers.CopyValidation.cs" />
-    <Compile Include="$(CommonPath)System\Net\Security\SslClientAuthenticationOptionsExtensions.cs"
-             Link="Common\System\Net\Security\SslClientAuthenticationOptionsExtensions.cs" />
-    <Compile Include="$(CommonPath)System\Net\NetworkErrorHelper.cs"
-             Link="Common\System\Net\NetworkErrorHelper.cs" />
-    <Compile Include="$(CommonPath)System\IO\DelegatingStream.cs"
-             Link="Common\System\IO\DelegatingStream.cs" />
-    <Compile Include="$(CommonPath)System\IO\ReadOnlyMemoryStream.cs"
-             Link="Common\System\IO\ReadOnlyMemoryStream.cs" />
-    <Compile Include="$(CommonPath)System\Text\StringBuilderCache.cs"
-             Link="Common\System\Text\StringBuilderCache.cs" />
-    <Compile Include="$(CommonPath)System\Net\Logging\NetEventSource.Common.cs"
-             Link="Common\System\Net\Logging\NetEventSource.Common.cs" />
-    <Compile Include="$(CommonPath)System\Net\HttpDateParser.cs"
-             Link="Common\System\Net\HttpDateParser.cs" />
-    <Compile Include="$(CommonPath)System\Text\SimpleRegex.cs"
-             Link="Common\System\Text\SimpleRegex.cs" />
-    <Compile Include="$(CommonPath)System\HexConverter.cs"
-             Link="Common\System\HexConverter.cs" />
-    <Compile Include="$(CommonPath)System\Net\ArrayBuffer.cs"
-             Link="Common\System\Net\ArrayBuffer.cs"/>
+    <Compile Include="$(CommonPath)System\IO\StreamHelpers.CopyValidation.cs" Link="Common\System\IO\StreamHelpers.CopyValidation.cs" />
```

Comment:
Could you split the line at `Link` attribute as it was? So that it's visible what's really changed.

Same with other csproj files.

Reply:
Yeah, this is VS being annoying and reformatting the whole file. I'll fix it.